src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaExtensions.cs

```diff
@@ -32,12 +32,12 @@ public static bool MatchesKind(this SyntaxTrivia trivia, params SyntaxKind[] kin
 
         public static bool IsRegularComment(this SyntaxTrivia trivia)
         {
-            return trivia.IsSingleLineComment() || trivia.IsMultiLineComment();
+            return trivia.IsSingleLineComment() || trivia.IsMultiLineComment() || trivia.IsShebangComment();
```

Comment:
It's treated just like a single-line comment everywhere from a (compiler/language perspective), hence the name.  Originally, it didn't even have it's own ```SyntaxKind```, but I added that after it was suggested that hosts might want to easily distinguish these without comparing the leading ```char```.